src/components/Form/Input/Input.tsx

```diff
@@ -10,6 +10,8 @@ import NumberInput from './InputNumber'
 
 export interface GenericInputProps extends InputProps {
   controllers?: boolean
+  min?: number
```

Comment:
This props exist on `HTMLInputElement` by default no?

Here the example 

![imagen](https://github.com/dd3tech/dd360-components/assets/64227164/ca2d2d53-ebd8-422f-a602-e7e0f0efe546)